### DISCUSSION TITLE
How can find the all available `vim::Commands`?

### DISCUSSION THREAD

**Discussion by baiguoname** [zed-industries/zed#16148] [Feature Requests]
There are some vim commands like this:
```
vim::Matching,
vim::RepeatedFind,
```
How can I find the all available commands?
For example, if I need to map `1` to `!`, I can do this:
```
"1": ["workspace::SendKeystrokes", "!"],
```
But I thought there would be better way like:
```
"1": ["vim::PushString", "!"]
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
